package.json

```diff
@@ -22,11 +22,11 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "file:toolkit/actions-core-0.0.0.tgz",
-    "@actions/exec": "file:toolkit/actions-exec-0.0.0.tgz",
-    "@actions/github": "file:toolkit/actions-github-0.0.0.tgz",
-    "@actions/io": "file:toolkit/actions-io-0.0.0.tgz",
-    "@actions/tool-cache": "file:toolkit/actions-tool-cache-0.0.0.tgz",
+    "@actions/core": "^1.0.0",
+    "@actions/exec": "^1.0.0",
```

Comment:
Can you actually just cut exec, io, and tool-cache? We don't need them in this action,

Reply:
Extra packages removed in 8bc948481e24a0a64db20c4677dc7ba203481511